src/adapter.ts

```diff
@@ -262,4 +275,27 @@ export default class JestTestAdapter implements TestAdapter {
   private retireAllTests() {
     this.retireEmitter.fire({});
   }
+
+  private logSuite(suiteOrTest?: TestSuiteInfo|TestInfo, depth: number = 0): void {
+    if (suiteOrTest == null) { return; }
+
+    const indent = (inDepth: number): string => {
+      let spaces = "";
+      for (let i = 0; i < inDepth; i++) {
+        spaces += "  ";
+      }
+      return spaces;
+    };
+
+    if (suiteOrTest.label) {
+      this.log.info(`${indent(depth)}${suiteOrTest.label} (${suiteOrTest.type})`);
+    }
+
+    const suite = suiteOrTest as TestSuiteInfo;
+    if (suite.children) {
```

Comment:
See my other comment about what I have seen about truthy/falsy values in the project.

```suggestion
    if (suite.children !== 0) {
```